lib/engine/game/g_1880/step/buy_train.rb

```diff
@@ -79,8 +79,7 @@ def owns_restored2?(entity)
           def can_buy_restored2?(entity)
             @game.phase.available?(restored2_train&.available_on) &&
             !owns_restored2?(entity) &&
-            @corporations_sold.empty? &&
-            entity.cash >= restored2_train.price
+            @corporations_sold.empty?
```

Comment:
Will this change affect EMR? Can you check if this change lets you buy a 2P with player cash?

Reply:
My hunch is we'll need to remove depot owned 2Ps if the corp has less cash than they cost.

Reply:
i'm not clear about the rules:
1- can you cross buy a 2p when in EMR? (yes?)
2- can you sell shares in EMR, buy a 2p from the bank, then sell more shares to buy a regular train? 
3- can you sell shares in EMR then cross buy? (no?)

Reply:
Under forced train purchase
> The train acquired by Forced Purchase may not be a restored 2-train. 

No other special rules around restored 2-train, except that it doesn't count towards your train requirement

Reply:
So
1 - yes, provided it is with company cash
2 - no
3 - no, not for any train. Must be from the depot

If a company has no trains, but 250 in cash, it can buy a reclaimed 2-train and then EMR a train from the depot.

Reply:
much clearer now, yeah, i'll need to remove 2p from depot if low on cash

Reply:
Fixed. Maybe there's a better/cleaner way to code it?
I can wrap my head around the if/unless conditionals to make it work in one block